OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+
+```
+cd /scratch/docker-images/OracleWebCenterContent/dockerfiles
+sh buildDockerImage.sh -v 12.2.1.4.0
+
+#Verify you now have the image
+docker images
+```
+
+# 5. Running Oracle WebCenter Content Docker Container
+ 
+To run the Oracle WebCenter Content Docker container, you need to create:
+* Container to manage the Admin Server.
+* Container to manage the Managed Servers.
+
+## 5.1. Creating containers for WebCenter Content Server
+
+### 5.1.1. Update the environment file 
+
+Create an environment file `webcenter.env.list` file, to define the parameters.
+
+Update the parameters inside `webcenter.env.list` as per your local setup & place it under /scratch/wccdocker/ .
+Please note - all parameters mentioned below are manadatory and shouldn't be omitted or, left blank. 
+
+```
+#Database Configuration
+DB_DROP_AND_CREATE=<true or false>
+DB_CONNECTION_STRING=<Hostname/ContainerName>:<Database Port>:<Database Service>
+DB_RCUPREFIX=<RCU Prefix>
+DB_PASSWORD=<Database Password>
+DB_SCHEMA_PASSWORD=<Schema Password>
+
+#configure container
+ADMIN_SERVER_CONTAINER_NAME=<Admin Server Container Name>
+ADMIN_PORT=<Admin Server Port>
+ADMIN_PASSWORD=<Admin Server Password>
+ADMIN_USERNAME=<Admin Server User Name>
+
+DOMAIN_NAME=<domain directory-name>
+UCM_PORT=<port to be used for UCM managed server on container>
+IBR_PORT=<port to be used for IBR managed server on container>
+UCM_HOST_PORT=<host port to access UCM managed server - this is the port value to be used for -p option (left of the colon) sec. ### 5.1.3>
+IBR_HOST_PORT=<host port to access IBR managed server - this is the port value to be used for -p option (left of the colon) sec. ### 5.1.3>
+UCM_INTRADOC_PORT=<UCM intradoc port on container>
+IBR_INTRADOC_PORT=<IBR intradoc port on container>
+
+#component
+component=IPM,Capture,ADFUI
+
+#HOSTNAME
+HOSTNAME=<provide your host name>
+
+#Keep Container alive
+KEEP_CONTAINER_ALIVE=true
+```
+
+### 5.1.2. Admin Container (WCCAdminContainer)
+#### A. Creating and Running Admin Container
+
+Run the following command to create the Admin Server container:
+
+```
+docker run -it --name WCCAdminContainer --network=WCContentNET -p <Any Free Port>:<ADMIN_PORT> -v /scratch/wccdocker/volumes/wccontent:/u01/oracle/user_projects --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/createDomainandStartAdmin.sh; /bin/bash"
+
+# A sample command will look ike this -
+
+docker run -it --name WCCAdminContainer --network=WCContentNET -p 7001:7001 -v /scratch/wccdocker/volumes/wccontent:/u01/oracle/user_projects --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/createDomainandStartAdmin.sh; /bin/bash"
+```
+**Note:** The above command deletes any previous RCU with the same prefix if **DB_DROP_AND_CREATE=true**
+
+The docker run command creates the container as well as starts the Admin Server in sequence given below:
```

Comment:
Please wrap "docker run" in back ticks to format it with a monospaced font to indicate it is an actual command.